src/background.js

```diff
@@ -265,7 +310,7 @@ const rollout = {
     log("calling init");
 
     // Check if the add-on has run before
-    let doneFirstRun = await this.getSetting("doneFirstRun");
+    let doneFirstRun = await this.getSetting("doh-rollout.doneFirstRun");
```

Comment:
for users who are upgrading from v1.0.4, should we also check for the unprefixed doneFirstRun setting here?

If users who were stranded with TRR enabled cleared local storage, will we decide that they enabled TRR on their own and fail to run?

Reply:
@tdsmith The only two local storage items that are updated will be `doh-rollout.skipHeuristicsCheck` and `doh-rollout.doneFirstRun`. I think both of these can be omitted for the stranded users. The only local storage item of real significance for users who opted out is `doh-rollout.disable-heuristics`, which was already name spaced as such.

Reply:
My concern is that, once we enable this addon for users that were stranded with TRR enabled:
- Stranded users will not have any stored values for `doh-rollout.doneFirstRun` because that's not the key that v1.0.4 used.
- Therefore, the first-run check will run.
- The first-run check calls `trrModePrefHasUserValue`, which will observe that stranded users have a user-branch value for network.trr.mode.
- The addon will disable itself.

Unless I'm missing something, we'll end up back where we started.

Reply:
@tdsmith Oh goodness1 I see what you mean. Let me confirm.

Reply:
@tdsmith @nhnt11 Added simple migration function/local to address this issue. However, the easier method would be to revert the name change entirely for this release. Thoughts?

Reply:
It worries me that this migration is untested. From that perspective, fewer changes are probably better. The point of the prefix was for namespacing when we set prefs, right? Maybe it's enough to add/remove the namespacing when setting/getting prefs?

Anyway, I think we should go for the least added complexity, but we can't set prefs that aren't namespaced.

Reply:
Agreed. Both of these prefs are local storage only. I will roll these two items back and file and issue. Again – trying to touch a least code as possible.

Reply:
Rolled back and moved code to #175